internal/cmdupdate/cmdupdate.go

```diff
@@ -88,9 +90,12 @@ func (r *Runner) preRunE(_ *cobra.Command, args []string) error {
 		return err
 	}
 
-	relPath := string(p.DisplayPath)
 	r.Update.FullPackagePath = string(p.UniquePath)
 
+	relPath, err := resolveRelPath(p.UniquePath)
```

Comment:
Can you add a TODO to move the logic in 95-98 (including resolveRelPath) to Package.

Reply:
done